src/components/LoanCards/Buttons/LendIncrementButton.vue

```diff
@@ -18,15 +18,20 @@
 			:price="selectedOption"
 			:loan-id="loanId"
 			:loading.sync="loading"
+			@click.native="triggerAddToBasketInterstitial(loanId)"
 		/>
 	</div>
 </template>
 
 <script>
 import numeral from 'numeral';
 import LendButton from '@/components/LoanCards/Buttons/LendButton';
+import addToBasketInsterstitial from '@/plugins/add-to-basket-show-interstitial';
 
 export default {
+	mixins: [
+		addToBasketInsterstitial
```

Comment:
Understanding this is currently a WIP, could this become an event that bubbles up and is handled by `ActionButton` in the future?

Reply:
Definitely planning to add a loading sequence as I shore up the image + structure of the loan preview.

I'll also work to cleanup the duplication with that event, good call.